client/my-sites/plans/plans-features/google-analytics.js

```diff
@@ -33,7 +33,12 @@ export default function( context ) {
 
 	function goBack() {
-		page.back( '/plans/' + ( context.params.domain || '' ) );
+		const fallback = '/plans/' + ( context.params.domain || '' );
+		if ( ! context.prevPath || context.prevPath.indexOf( context.params.domain ) === -1 ) {
```

Comment:
instead of the if else, we should be able to simplify this to:

```javascript
page.back( fallback );
```

Page.js navigates to your fallback link if your history is empty:
https://github.com/visionmedia/page.js/blob/master/page.js#L224